### PR TITLE
EPMRPP-76491 || project settings header

### PR DESCRIPTION
**Pull request by chivekrodis**

### CODE REVIEW

**Review by tr1ble — APPROVED** [review]

**Review by Vadim73i — APPROVED** [review]

File: app/src/pages/inside/projectSettingsPageContainer/projectSettingsPageContainer.jsx [hunk 1]
```diff
@@ -166,10 +167,17 @@ export const ProjectSettingsPageContainer = () => {
     return itemsConfig[activeTab].component;
   }, [activeTab, createItemsConfig, dispatch]);
 
+  const header = useMemo(() => {
```
Comment (AmsterGet) [inline app/src/pages/inside/projectSettingsPageContainer/projectSettingsPageContainer.jsx:170]: Is it necessary to memo such plain component?
You don't drill down it as a prop and it doesn't contain any calculations.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/projectSettingsPageContainer/projectSettingsPageContainer.jsx [hunk 2]
```diff
@@ -164,17 +164,16 @@ export const ProjectSettingsPageContainer = () => {
...
 
     return <Header title={title} />;
-  }, [createItemsConfig, activeTab]);
+  };
 
   return (
     <SettingsLayout navigation={navigation}>
       <ScrollWrapper>
-        <div className={cx('header')}>{header}</div>
+        <div className={cx('header')}>{getHeader()}</div>
```
Comment (AmsterGet) [inline app/src/pages/inside/projectSettingsPageContainer/projectSettingsPageContainer.jsx:176]: Why not just render the `<Header title={config[activeTab].name} />`?

**Review by AmsterGet — COMMENTED** [review]
One more comment about `tabsConfig`, now we don't have tabs. Please rename to smth like `navConfig` or `pagesConfig` or smth like that.

**Review by AmsterGet — APPROVED** [review]

**Comment by AmsterGet** [comment]
Please resolve conflicts.

**Pull request merged**